Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser.yml

```diff
@@ -437,9 +443,72 @@ script:
     - contextPath: MSGraphUserDirectReports.Reports.UserPrincipalName
       description: The user principal name (UPN) of the user. The UPN is an Internet-style login name for the user based on the Internet standard RFC 822. By convention, this should map to the user's email name. The general format is alias@domain, where domain must be present in the tenant’s collection of verified domains. This property is required when a user is created. The verified domains for the tenant can be accessed from the verifiedDomains property of organization.
       type: String
-    description: Retrieves the direct reports for a user. Direct reports are the people
-        who have that user configured as their manager.
-  dockerimage: demisto/crypto:1.0.0.7634
+  - arguments:
+    - default: false
+      description: The User ID or userPrincipalName of the user for which to get the manager.
+      isArray: false
+      name: user
+      required: true
+      secret: false
+    deprecated: false
+    description: "Retrieves the properties from the manager of a user"
+    execution: false
+    name: msgraph-user-get-manager
+    outputs:
+    - contextPath: MSGraphUserManager.ID
+      description: User's ID.
+      type: Unknown
```

Comment:
please set the output types

Reply:
This part was mostly copied from other commands ;)
I added the output types in the latest commit